support/grunt/patches/index.js.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/examples/index.js b/examples/index.js
-index 907b324..b028b2e 100644
+index 02f4aef..a38b2b6 100644
```

Comment:
not sure what this does, but is strange to have a patch file committed (I'm sure there is a good reason for it, I just don't know what it is)

Reply:
It rewrites some paths in files for a distribution build until we have a full webpack bundle working which deals with paths.